nb2kg/handlers.py

```diff
@@ -40,6 +40,9 @@
 KG_CONNECT_TIMEOUT = float(os.getenv('KG_CONNECT_TIMEOUT', 20.0))
 KG_REQUEST_TIMEOUT = float(os.getenv('KG_REQUEST_TIMEOUT', 20.0))
 
+# Keepalive ping interval (30 seconds)
+WS_PING_INTERVAL_MS = 30000
```

Comment:
Could we make this value a function of a _seconds-based_ environment variable (e.g., `KG_WS_PING_INTERVAL_SECS`) that defaults to 30?  I'd like the option of being able to change this w/o building.

Reply:
I'll fix this as you recommended. BTW do you have any plan to move those settings from Environment Variable to Configuration File such as ini, yaml, or even `.py` dealing with `traits`?

Reply:
Thanks for the update.

No plans on creating formal config values for these since [NB2KG is embedded](https://github.com/jupyter/notebook/pull/4161) in Notebook (in master tip) where these have been promoted to formal config values (with env support).  Once NB 6 is released, there won't be a need to deal with a separate NB2KG server extension (although that approach would still work provided once doesn't start NB with `--gateway-url`).